docs/Introduction/whatIsLitProtocol.md

```diff
@@ -47,7 +47,7 @@ Lit’s access control protocol gives individuals the ability to **read** privat
 
 To facilitate this second use case we created two things: Lit Actions and Programmable Key Pairs (PKPs). Lit Actions are immutable JavaScript functions stored on IPFS. Actions can be thought of as smart contracts with superpowers: they have network access and the ability to make arbitrary HTTP requests. 
 
-PKPs are the public/private key pairs generated by the Lit network and are minted in the form of an ERC-721 NFT. Only the owner of the NFT can combine the underlying key shares and form the complete signature. 
+PKPs are the public/private key pairs generated by the Lit network and are minted in the form of an ERC-721 NFT. Only those with authorized access to sign with the PKP (via an [auth sig](/SDK/Explanation/WalletSigs/sessionSigs)) can combine the underlying key shares.
```

Comment:
Comment from below - would remove via an auth sig. The full explanation for how someone might authorize access can be a later update / its own section later.